addons/account/views/audit_trail_report_views.xml

```diff
@@ -27,7 +27,11 @@
                 <field name="account_audit_log_move_id"/>
                 <field name="author_id"/>
                 <field name="date" string="Date"/>
-                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
+                <filter string="Updated Entries" name="audit_updated_entries" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
```

Comment:
Note that it could be the opportunity to rename the file to ``mail_message_views.xml`` as what matters is the content, allowing to discover the models and views reading file names (and opening all mail_message_views.xml files allows to check a bit what are the various usage of message views).